tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -41,6 +41,45 @@
 def histogram(name, data, step=None, buckets=None, description=None):
     """Write a histogram summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Writes a histogram to the current default summary writer, for later analysis
+    in TensorBoard's 'Histograms' and 'Distributions' dashboards (data written
+    using this API will appear in both places). Like `tf.summary.scalar` points,
+    each histogram is associated with a `step` and a `name`. All the histograms
+    with the same `name` constitute a time series of histograms.
+
+    When higher order `Tensor`s are flattened to produce a histogram of all
+    entries within.
+
+    This example writes 2 histograms:
+
+    ```python
+    w = tf.summary.create_file_writer('test/logs')
```

Comment:
Is it possible to make this a testable example?  See https://www.tensorflow.org/community/contribute/docs_ref

Reply:
(response copied from https://github.com/tensorflow/tensorboard/pull/4795)

I updated it to use "doctest style" `>>>`, `...`s, but I can't seem to get the test to run at all.

Digging into `bazel run //tensorflow/tools/docs:tf_doctest`, it seems that passing `--file` only works when the file is in the tensorflow repo, which TensorBoard's V2 summaries are not.  Passing `--module` and building from source didn't work for me, as I got some unrelated mlir error during the bazel build.

I did do a test sync cl/364023724, which runs an TAP presubmit.  Is this sufficient?

Reply:
Similar to the tf.summary.image() PR, I've reverted the doctest style for now, with the intent of keeping these PRs narrowly scoped.  Introducing a doctest process in our workflow sounds like a good, separate task.

Reply:
For posterity, b/183515215 tracks the internal effort to disentangle the tf.summary API from the TensorBoard repo, which might make the tensorflow/ repo the natural place for these doctests in the future.